src/assets/scripts/client/airline/AirlineModel.js

```diff
@@ -205,42 +194,11 @@ export default class AirlineModel extends BaseModel {
         return this._getRandomAircraftTypeFromFleet(fleet);
     }
 
-    // TODO: the logic here can be simplified.
-    /**
```

Comment:
we still need this docblock.  If you could add it back in, I think we'll be good to go here.